src/env.ts

```diff
@@ -40,3 +40,6 @@ export const TS_BLUE = '#007ACC';
 export const GREEN = '#77b155';
 // Picked from Discord's "hourglass" emoji (in ⌛ | Occupied Help Channels)
 export const HOURGLASS_ORANGE = '#ffa647';
+// Picked from Discord's blockquote line
+export const BLOCKQUOTE_GREY = '#4f545c';
```

Comment:
I'm pretty sure this is unused. Also, defining the same thing twice in a different format is a bit wonk as I believe D.js normalizes arguments anyway.

Reply:
The place I'm using `BLOCKQUOTE_GREY` is required to be a number. I can remove the unused `BLOCKQUOTE_GREY` constant and/or change `BLOCKQUOTE_GREY_NUMBER` to `BLOCKQUOTE_GREY` if desired.

Reply:
Would it be better to drop `BLOCKQUOTE_GREY_NUMBER` and use `parseInt(BLOCKQUOTE_GREY.slice(1), 16)`?

Reply:
Sure, that works too.